methods/methods_release2.md

```diff
@@ -93,49 +95,262 @@ Demultiplexing, feature-table generation, fragment insertion, and taxonomic prof
+  --i-table emp500_18s_deblur_merged_biom_euks.qza \
+  --p-min-samples 2 \
+  --o-filtered-table emp500_18s_deblur_merged_biom_euks_noSingletons.qza`
+
+#### Filtering control samples
+`qiime feature-table filter-samples \
+  --i-table emp500_18s_deblur_merged_biom_euks_noSingletons.qza \
+  --m-metadata-file emp500_metadata_basic.txt \
+  --p-where 'empo_1 != "Control"' \
+  --o-filtered-table emp500_18s_deblur_merged_biom_euks_noSingletons_noControls.qza` 
+
+#### Filtering samples with low read counts
+`qiime feature-table filter-samples \
+  --i-table emp500_18s_deblur_merged_biom_euks_noSingletons_noControls.qza \
+  --p-min-frequency 5500 \
+  --o-filtered-table emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min5500.qza` 
+
+#### Estimate beta-diversity
+`qiime deicode rpca \
+  --i-table emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min3500.qza \
+  --p-min-feature-count 0 \
+  --p-min-feature-frequency 10 \
+  --p-min-sample-count 0 \
+  --o-biplot emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min5500_rpca_pca.qza \
+  --o-distance-matrix emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min5500_rpca_dist.qza`
 
 ### 1.3 Fungal ITS data
 
 #### Demultiplexing:
-* Process: Split libraries FASTQ
-** Parameters: Multiplexed FASTQ, Golay 12 base pair reverse complement mapping file barcodes
+`qiime demux emp-paired \
+  --i-seqs emp-paired-end-sequences.qza \
+  --m-barcodes-file emp500_prep_info_ITS.txt \
+  --m-barcodes-column 'barcode' \
+  --p-rev-comp-mapping-barcodes \
+  --o-error-correction-details emp500_its_demux.qza \
+  --o-per-sample-sequences emp500_its_demux.qza`
+
+#### Trimming primers:
+`qiime cutadapt trim-paired \
+  --i-demultiplexed-sequences emp500_its_demux.qza \
+  --p-adapter-f GCATCGATGAAGAACGCAGC \
+  --p-front-r GCTGCGTTCTTCATCGATGC \
+  --o-trimmed-sequences emp500_its_demux_trimmed.qza`
+  
+#### Export data and process for denoising with Deblur (standalone)
+Note: The sample list is a text file where each row is a unique sample identifier that corresponds to your sequence file names. For example, 'uren.s001' corresponds to the sequence files 'uren.s001_S1_L001_R1.fastq.gz' and 'uren.s001_S1_L001_R2.fastq.gz'.
 
-#### Sequence trimming, denoising, feature-table generation, and fragment insertion:
-* Process: Trimming
-* Parameters: 150 base pair
+`qiime tools export \
+  --input-path emp500_its_demux_trimmed.qza \
+  --output-path emp500_its_demux_trimmed`
+  
+`input_path='emp500_its_demux_trimmed/'`
+`output_path='emp500_its_demux_trimmed_merged_files/'`
+`sample_list='sample_list.txt'`
 
-* Process: Deblur
-* Parameters: Default settings
+`for i in $(cat < "$sample_list");
+do
+cat "$input_path""$i"*R1*fastq.gz "$input_path""$i"*R2*fastq.gz > "$output_path""$i".fastq.gz
+done`
 
-#### Taxonomic profiling
-* Process: Feature-classifier sklearn 
-* Parameters: Using the UNITE 8 release as a reference
+#### Denoising
+`deblur workflow \
```

Comment:
here and in the 18S workflow, should `--min-reads` be set to 1? it's default is 10, not sure if that is desired or not

Reply:
Thanks! We applied the default study-wide filter of --min-reads = 10, so should report that. I'm curious whether deblur in Qiita also has this default?

Reply:
OK thanks. I'm going to include this flag as a recommendation to avoid study specific bias.